src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -35,19 +39,37 @@ trainer = pl.Trainer(logger=AimLogger(experiment='experiment_name'))
     {
       title: 'Integrate Hugging Face',
       docsLink: DOCUMENTATIONS.INTEGRATIONS.HUGGING_FACE,
-      code: `from aim.hugging_face import AimCallback
-
-# ...
-aim_callback = AimCallback(repo='/path/to/logs/dir', experiment='mnli')
-trainer = Trainer(
-    model=model,
-    args=training_args,
-    train_dataset=train_dataset if training_args.do_train else None,
-    eval_dataset=eval_dataset if training_args.do_eval else None,
-    callbacks=[aim_callback],
-    # ...
-)
-# ...`,
+      code: `import mlflow
+import transformers
+      
+      
+def create_huggingface_model(model_path):
+    architecture = "lordtt13/emo-mobilebert"
+    mlflow.transformers.save_model(
+        transformers_model={
+            "model": transformers.TFMobileBertForSequenceClassification.from_pretrained(
+                architecture
+            ),
+            "tokenizer": transformers.AutoTokenizer.from_pretrained(architecture),
+        },
+        path=model_path,
+    )
+    llm = mlflow.transformers.load_model(model_path)
+    prompt = PromptTemplate(
+        input_variables=["product"],
+        template="What is a good name for a company that makes {product}?",
+    )
+    hf_pipe = HuggingFacePipeline(pipeline=llm)
+    return LLMChain(llm=hf_pipe, prompt=prompt)
+
+
+mlflow.set_tracking_uri("${fasttrack_server}")
```

Comment:
maybe the `set_tracking_uri` invocation should go at top of example right under the imports section, since that is our key difference